src/transport/TaskManagerGRPCService.cpp

```diff
@@ -0,0 +1,136 @@
+//
+// Created by Anton Ovcharenko on 25.01.2022.
+//
+
+#include "TaskManagerGRPCService.h"
+
+TaskManagerGRPCService::TaskManagerGRPCService(const std::shared_ptr<ModelInterface> &model) :
```

Comment:
Doesn't look like renaming

Reply:
It is -- renaming of the file, GitHub marks `src/transfer/TaskManagerService.cpp` as deleted (see above) and `src/transport/TaskManagerGRPCService.cpp` as newly added, even though I just changed the file and directory names